### DISCUSSION TITLE
Switching active bootbank on TG799VAC vant-r

### DISCUSSION THREAD

**Discussion by mojja** [hack-technicolor/hack-technicolor#166] [Q&A]
Been trying to swap the active bootbank for over 2 hours now with the Crazy Power switching method. I just can't get it to work. I have tried every timing. Am i doing it wrong or is it just extremly difficult to pull off?

Im basically doing the same thing as the video linked. Only thing im changing are the timings, and yes i have tried with the same timings as in the video.

**Comment by LuKePicci** [comment]
I'd say it's extremely difficult to do it right, but who does it at least once then he generally knows how to do it over and over again easily.

**Reply by mojja** [reply]
Okay thanks, I guess I’ll just keep on trying then. Wanted to make sure it’s even possible to pull off before i waste more hours on it.
